src/coreclr/tools/aot/crossgen2/crossgen2_publish.csproj

```diff
@@ -18,11 +18,8 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <NativeAotSupported Condition="$(OutputRID.StartsWith('tizen')) == 'true'">false</NativeAotSupported>
-    <NativeAotSupported Condition="$(OutputRID.EndsWith('-arm')) == 'true'">false</NativeAotSupported>
+    <!-- block win-x86 PublishAot until preview 5 -->
```

Comment:
> preview 5

or 6, depending on if https://github.com/dotnet/sdk/pull/41130 is merged in time before the branding PR. 😅

Reply:
> > preview 5
> 
> or 6, depending on if [dotnet/sdk#41130](https://github.com/dotnet/sdk/pull/41130) is merged in time before the preview 6 branding PR. 😅

The SDK PR is a good change, but the runtime pack is only used if `PublishAotUsingRuntimePack==true` and we only need that for apple/bionic. I would be surprised if that blocks anything for this.

Reply:
The CI run from previous commit was giving me:

> .dotnet\sdk\9.0.100-preview.3.24204.13\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(90,5): error NETSDK1203: (NETCORE_ENGINEERING_TELEMETRY=Restore) Ahead-of-time compilation is not supported for the target runtime identifier 'win-x86'.

This (publish-only) filtering seem to have done the trick.

Reply:
win-x86 didn't make the Preview 3 cutoff. This error should be gone in preview 4.

Also, I'm actually not sure in what shape Linux-arm32 is in Preview 3 and we don't have any testing for arm32 hosted ILC so I'd rather keep the blocking until preview 4.

Reply:
Ah, for LA64 that was the reason why I created this PR only _after_ https://github.com/dotnet/runtime/pull/102604. So great that x86 will be resolved as well if we wait that one to go in first. Shall remove this workaround. 👍

Reply:
Removed the x86 workaround and merged main (which is on preview4).